ui/selectors/multichain-accounts/account-tree.ts

```diff
@@ -193,6 +194,38 @@ export const getWalletsWithAccounts = createDeepEqualSelector(
   },
 );
 
+/**
+ * Retrieve the normalized groups metadata.
+ *
+ * @param accountTree - Account tree state.
+ * @param internalAccountsObject - The internal accounts object.
+ * @returns The normalized group metadata.
+ */
+export const getNormalizedGroupsMetadata = createDeepEqualSelector(
```

Comment:
Added a selector that gets you the pertinent group search metadata that has already been lower cased, so that we're not repeatedly doing this during a search operation.